src/components/common/DocumentDelivery.vue

```diff
@@ -41,6 +41,15 @@
         <span id="custodian-email">{{custodianEmail || '(Not entered)'}}</span>
       </v-col>
     </v-row>
+
```

Comment:
Do you need a block for Business Contact also?

<img width="997" alt="Screen Shot 2022-03-18 at 9 51 14 AM" src="https://user-images.githubusercontent.com/10002889/159060950-650b9a26-6f85-4b9f-ba0d-f001db4f7ff6.png">

Reply:
You are right , we need it, I was thinking can I use the same registeredOfficeEmail field for value and label convert to prop?
Like ```@Prop({ default: 'Registered Office' }) readonly registeredOfficeLabel: string```
Or do you prefer to add it as an additional prop (may be businessContactEmail) and put v-if ?

Reply:
Thats a good idea i think!
If you are going to change the label to allow variations, a more generic prop name might be better, even though you'd default to `Registered Office`. `OfficeLabel` or something maybe? We are moving towards variations of offices, ie registeredOffice, recordsOffice, businessOffice etc

Reply:
1. Yes, let's do something generic for the first address. I just noticed that in my feature, I also need to change the first label! Can we call it `contactLabel` and `contactValue` (or contactEmail) ?

1, Don't use a default value as that can lead to cascading properties, which will be hard to figure out in the future. Instead, update everywhere this is used to pass in the desired label.